std/random.d

```diff
@@ -1443,10 +1566,19 @@ alias Xorshift    = Xorshift128;                            /// ditto
         [5783321UL, 393427209, 1947109840, 565829276, 1006220149, 971147905,
         1436324242, 2800460115, 1484058076, 3823330032],
         [0UL, 246875399, 3690007200, 1264581005, 3906711041, 1866187943, 2481925219,
```

Comment:
Pre-existing unittests test the output of `XorshiftEngine` and verify the sequence produced has not changed.